src/jit/emit.cpp

```diff
@@ -3000,6 +3003,10 @@ emitter::instrDesc  * emitter::emitNewInstrCallInd(int        argCnt,
         id->idcArgCnt         = argCnt;
         id->idcDisp           = disp;
 
+#if defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
```

Comment:
We should avoid UNIX_AMD64_STRUCT_PASSING #fdefs in emitter. Rather it would be good to expose it as a feature by emitter to support two register returns. e.g. FEATURE_MULTIREG_RETURN